composer.json

```diff
@@ -199,7 +199,7 @@
         "behat/behat": "^3.6.1",
         "behat/mink-selenium2-driver": "^1.4",
         "dmore/behat-chrome-extension": "^1.3",
-        "dmore/chrome-mink-driver": "^2.7",
```

Comment:
What happended in `2.8.1-beta1` version that we have to update it?

Reply:
There was a `Child process error` that kept coming, as you can see now and I discovered that the solution for this problem is here: https://gitlab.com/DMore/chrome-mink-driver/-/merge_requests/111
That was just a try if it  fixes that and it does, but it isn't probably the best way.